File: py-sellergoods-interface/src/main/java/com/kevin/sellergoods/service/BrandService.java
package com.kevin.sellergoods.service;

import com.kevin.pojo.TbBrand;
import com.vo.PageBean;

import java.util.List;

/**
 * Created by IntelliJ IDEA.
 * User: kevin
 * Date: 2018/6/27
 * 品牌接口
 */
public interface BrandService {
    public List<TbBrand> findAll();

    //品牌分页
    public PageBean findPage(int currPage, int pageSize);

    public void add(TbBrand tbBrand);

    public TbBrand findOne(Long id);

    public void update(TbBrand tbBrand);

    public void delete(Long[] ids);

    public PageBean findPage(TbBrand tbBrand, int currPage, int pageSize);


}

File: py-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {
    this.findAll = function () {
        return $http.get('/brand/findAll');
    };

    this.findPage = function (page, size) {
        return $http.get("/brand/findPage?currPage=" + page + '&pageSize=' + size);
    };

    this.findOne = function (id) {
        return $http.get('/brand/findOne?id=' + id);
    };
    this.add = function (entity) {
        return $http.post('/brand/add', entity);
    };
    this.update = function (entity) {
        return $http.post('/brand/update', entity);
    };
    this.dele = function (ids) {
        return $http.get('/brand/delete?ids=' + ids);
    };
    this.search = function (page, size) {
        $http.post("/brand/search?currPage=" + page + '&pageSize=' + size, $scope.searcEntity);
    };
});